hydra-core/lib/generators/hydra/templates/catalog_controller.rb

```diff
@@ -3,9 +3,7 @@
 
 class CatalogController < ApplicationController
 
-  include Blacklight::Catalog
-  include Hydra::Controller::ControllerBehavior
-  include Hydra::Controller::SearchBuilder
+  include Hydra::Catalog
```

Comment:
I don't see where Hydra::Catalog also includes Hydra::Controller::ControllerBehavior, or do we want those separate?